lib/Parser.php

```diff
@@ -272,6 +268,10 @@ public function includeFiles(string $document) : string
             function ($match) {
                 $path = $this->environment->absoluteRelativePath($match[1]);
 
+                if (! is_readable($path)) {
```

Comment:
What's the difference between throwing in the negative scenario of `includeFileAllowed` returning `false` and an explicit `is_readabl()` here?

Also, if inclusion is disabled, does `is_readable()` trigger stream wrappers?